### ISSUE TITLE
Docker build works / run fails

### ISSUE THREAD

**Issue by jmcparland** [flowroute/sms-proxy#1]
I've tried the instructions with Ubuntu server 18.xx and Centos 7 as a base OS (in AWS).  The docker build works without a hitch, but I'm seeing the following two errors on docker run:

```
docker run -p 6644:8000 sms_proxy:0.0.1
Error loading shared library libpython2.7.so.1.0: No such file or directory (needed by /app/ve/bin/gunicorn)
Error relocating /app/ve/bin/gunicorn: Py_Main: symbol not found
```
Since I'm using your code as an intro to Docker example as well and since I've not previously used alpine linux, I'm not exactly sure how to look inside the exited container for clues. I appreciate the code example (I'll try getting it running in an ordinary vm) and am not expecting you to worry about a three-year-old example, but I'll watch in case there's a quick fix :-)   Thanks!

**Comment by jmcparland** [comment]
Moving package "python-dev" from the "apk --virtual" block to the block above it in the Dockerfile seems to clear the "docker run" error surrounding libpython2.7.so , allowing a clean start.

**Comment by hjhixon** [comment]
Good catch; I found in my testing that pinning alpine to 3.4 works as well. Glad to see you found a solution, I'll test your PR and we should be good to merge it.

**Comment by jmcparland** [comment]
Thanks, Harry! Use whatever solution works best, of course ~ I didn't go so far as to check what other remnants that package might throw into the container. Regardless, it's a really nice exemplar for the Flowroute API, Flask, Docker, python style, etc. ~ I'm really glad the crew shared it. Thanks again!
